bigframes/pandas/__init__.py

```diff
@@ -598,6 +599,18 @@ def read_gbq_table(
 read_gbq_table.__doc__ = inspect.getdoc(bigframes.session.Session.read_gbq_table)
 
 
+def read_gbq_table_streaming(table: str) -> bigframes.streaming.StreamingDataFrame:
```

Comment:
Could we move this to `bigframes.streaming.read_gbq_table`?

I think the Session method is OK for now, though I wonder if we would want a separate Session for streaming contexts? Session has a lot of configuration and implementation specific to pandas emulation.

Reply:
Done